### DISCUSSION TITLE
Pin badges

### DISCUSSION THREAD

**Discussion by hearnsj** [hpc-social/hpc-social.github.io#42] [General]
I am attending the Computing Insights UK conference on 1/2 December.  It is a UK HPC Conference.
https://www.scd.stfc.ac.uk/Pages/CIUK2022.aspx

I ordered up 100 pin badges with the HPC.social logo. I hope this is OK
Assuming the badges arrive in time I will make the available on the Dell stand - i must stress this is not from Dell and I paid for these out of my pocket.
I have addressed the badges direct to the conference centre - here's hoping they arrive.

**Comment by vsoch** [comment]
Super cool! Can you share a picture when they are arrive?

**Reply by alansill** [reply]
Great idea! I've been to that conference and it's a great event. 

I have a design for stickers also with the "Meet your colleagues in a new environment" tag line. I looked into getting these made for SC22 but at that point we were still negotiating with the hosting provider and doing tests. 

I'm working on organizing an official sponsorship page and mechanism but meanwhile this is a good approach. To facilitate this kind of thing we can put together a page of high quality graphics and some guidelines and rules for their use.

**Comment by alansill** [comment]
How did this go? Are there any pictures of the pins, and did you have many takers?

**Comment by hearnsj** [comment]
The package of pins was returned to sender... IT first went to Manchester United football club!
I have an image of the proofs on Stickermule but they make it hard to download the image.

If the pins ever do arrive I will update.

**Reply by hearnsj** [reply]
I now have 100 HPC.social pin badges!
I will be happy to send one free of charge to anyone in the UK.
DO we have ideas on getting them to other countries?

![IMG_20221216_103043](https://user-images.githubusercontent.com/19259589/208079946-d647cdee-a96c-494f-a362-eae13593dd76.jpg)

**Comment by alansill** [comment]
The FOSDEM HPC, Big Data, and Data Science devroom folks expressed interest in having something to hand out (pins like this, for example, or a flyer or sticker) for their [event in early February](https://fosdem.org/2023/schedule/track/hpc_big_data_and_data_science/). If you want to reuse these there, that is an option, or we can order a range of merch for such events if we can get a sponsor. Let me know what you think.

**Comment by hearnsj** [comment]
FOSDEM is a good idea. I know Kenneth Hoste and Adam Huffman. I should be there, but will contact Adam in case I cannot make it and will pass on the badges. I have 100 of them.

**Reply by alansill** [reply]
Thinking of what it would take to add merch to help with support. I could probably run this through one of the sponsoring organizations without too many complications. Still waiting for our GitHub sponsors application to be acted upon, but we could do something once it is there where we provide a pin or sticker as a thank-you for a supporting donation and still stay within the non-profit boundaries for our sponsors.

Iʻd like to have at minimum the dino icon, the hpc.social square or rectangular inverse logos, and maybe the Mastodon banner card as options. These woould each make nice pins or stickers, maybe even t-shirts with suitable design adjustments.

@vsoch do you have vector or other high-res originals for the dinosaur icon? We do have them for the rest.
